packages/js-client-rest/src/qdrant-client.ts

```diff
@@ -415,16 +463,18 @@ export class QdrantClient {
      * Count points in the collection matching the given filter.
      * @param collection_name
      * @param {object} args
-     *     - count_filter: filtering conditions
+     *     - shard_key: Specify in which shards to look for the points, if not specified - look in all shards
+     *     - filter: filtering conditions
      *     - exact:
      *         If `True` - provide the exact count of points matching the filter.
      *         If `False` - provide the approximate count of points matching the filter. Works faster.
      *         Default: `true`
      * @returns Amount of points in the collection matching the filter.
      */
-    async count(collection_name: string, {filter, exact = true}: SchemaFor<'CountRequest'> = {}) {
+    async count(collection_name: string, {shard_key, filter, exact = true}: SchemaFor<'CountRequest'> = {}) {
```

Comment:
Check if read_consistency is required. Grpc introduced `read_consistency` field in count request

Reply:
It's strange  for me, but count request does not have this parameter in openapi